src/msw/listctrl.cpp

```diff
@@ -3092,9 +3105,13 @@ bool HandleSubItemPrepaint(LPNMLVCUSTOMDRAW pLVCD, HFONT hfont, int colCount)
     HIMAGELIST himl = ListView_GetImageList(hwndList, LVSIL_SMALL);
     if ( himl && ImageList_GetImageCount(himl) )
     {
+        int wImage, hImage;
+        ImageList_GetIconSize(himl, &wImage, &hImage);
+
         if ( it.iImage != -1 )
         {
-            ImageList_Draw(himl, it.iImage, hdc, rc.left, rc.top,
+            int imgY = rc.top + ((rc.bottom - rc.top) / 2 - hImage / 2);
```

Comment:
Supernitpick: I would rename `imgY` to `yImage`, to be consistent with `wImage` and `hImage`.